examples/v1beta1/pipelineruns/4808-regression.yaml

```diff
@@ -92,4 +92,4 @@ spec:
     name: result-test
   params:
   - name: RESULT_STRING_LENGTH
-    value: "3000"
+    value: "2000"
```

Comment:
why changing this to 2000? will 3000 exceed the limit in test due to more data in termination msg?

Reply:
I agree. No need to change it.

Reply:
termination message allows up to 4096 bytes.